requirements/base.in

```diff
@@ -1,2 +1,3 @@
 click
 boto3
+attrs
```

Comment:
Newline issue again 😬

Reply:
Also, I don't see where `attrs` is used.

Reply:
It is imported in metric.py and used as an annotation. You added it 4 months ago...

Reply:
Newline added. You're killin' me smalls...

Reply:
Wow, thanks for fixing it.

Reply:
LOL